.github/workflows/pull_request.yaml

```diff
@@ -8,6 +8,8 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    env:
```

Comment:
V minor - If we're using the dev dummy key in the repository root here anyway, we could just modify the `Makefile` itself to be something like

```
test:
	NOTIFY_API_KEY=$$(cat ./dev-notify-api-key) pipenv run ./scripts/run_tests.sh
```

That way, we wouldn't need to add any env var config to the GA? I may have got the wrong end of the stick here though (sorry if so!)

Reply:
I added the env var config to GA in preparation for when this repo will eventually add the integration tests back and use an API key.  As both methods work I am happy to change if you would like 👍

Reply:
Ah sounds good! May as well keep it future-proofed ready for the int tests later on, so happy to keep it as it was originally 👍